ci/docs/build.sh

```diff
@@ -45,25 +45,25 @@ conda list --show-channel-urls
 
 # Build Doxygen docs
 gpuci_logger "Build Doxygen docs"
-cd $PROJECT_WORKSPACE/cpp/build
+cd "$PROJECT_WORKSPACE/cpp/build"
 make docs_cugraph
 	
 # Build Python docs
 gpuci_logger "Build Sphinx docs"
-cd $PROJECT_WORKSPACE/docs
+cd "$PROJECT_WORKSPACE/docs"
 make html
 
 #Commit to Website
-cd $DOCS_WORKSPACE
+cd "$DOCS_WORKSPACE"
 
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
         mkdir -p api/$PROJECT/$BRANCH_VERSION
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*	
+    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*"	
```

Comment:
I think the quotes here will prevent shell expansion from happening with the `*`. This may need to be tested locally. Maybe something like: `"$DOCS_WORKSPACE"/api/$PROJECT/$BRANCH_VERSION/*`, though I didn't test this.